Add sample plugin for Task API

RomanNikitenko opened the pull request:
The plugin demonstrates using of [the Task API](https://github.com/theia-ide/theia/blob/fc1edf11acf54f08f927686c567c0747e0d3644b/packages/plugin/src/theia.d.ts#L7201-L7221).

The short demo: https://youtu.be/Wfi5brEDojI

Signed-off-by: Roman Nikitenko <rnikiten@redhat.com>

benoitf commented:
hello, what will be the difference between this example and https://github.com/eclipse/che-theia/pull/97
because it seems that if it's almost the same goal then we don't need an example as we've a real-world example on che-theia

RomanNikitenko commented:
We use plugin API in ours plugins, so sure these ones are examples how to use API.
In my opinion the difference is:
- user may not even know about which plugin he can use as sample for using corresponding API - but 
 it's expected to search good example in che-theia-samples
- our plugins can contain a lot of logic vs for che-theia-samples case it's easier to find code you are interested in   
For current case I removed all che specific things, simplified as far as possible che-task plugin and represent it as sample. 
But I don't mind  - we can use che-task-plugin as sample, up to you.
